auditing_standards.md

```diff
@@ -218,15 +218,25 @@ Crates with this criteria contain unsafe Rust code which doesn't uphold the typi
     *   Writing implementations of traits not marked `unsafe` by violating documented invariants.
     *   Writing implementations of traits not marked `unsafe` that are not really intended to be implemented by the user.
     *   Explicitly forgetting values that have important drop behavior to cause UB when combined with operations that would not be expected to follow normally.
-*   A fair amount of caution may be required to avoid undefined behavior.
+*   Some caution may be required to avoid undefined behavior.
+
+
+`ub-risk-3` is expected to live at the threshold of acceptance as a dependency for most codebases: there is unsoundness in the crate, but there are also many situations where one need not worry about that unsoundness ever being triggered, and the codebase needs to make a judgement call. For example, one may choose to depend on a `ub-risk-3` dep if the rating is due to:
```

Comment:
```suggestion
`ub-risk-3` is the highest risk level at which non-experts could reasonably avoid causing undefined behavior. These crates are unsound, but the risk they pose may be acceptable in some situations. Projects need to make judgement calls about where, when, and by whom these crates may be used. For example, a project may deem a `ub-risk-3` crate acceptable to use if it contains unsoundness that:
```
Do you think that the lemma of "non-experts could reasonably avoid causing UB" is a useful one for determining `ub-risk-3`?